modules/web/src/de/diedavids/cuba/runtimediagnose/web/screens/console/ConsoleWindow.groovy

```diff
@@ -1,78 +1,130 @@
 package de.diedavids.cuba.runtimediagnose.web.screens.console
 
-import com.haulmont.cuba.gui.components.AbstractWindow
-import com.haulmont.cuba.gui.components.Frame
-import com.haulmont.cuba.gui.components.SourceCodeEditor
-import com.haulmont.cuba.gui.components.SplitPanel
-import de.diedavids.cuba.runtimediagnose.diagnose.DiagnoseExecution
+import com.haulmont.chile.core.model.MetaClass
+import com.haulmont.chile.core.model.MetaProperty
+import com.haulmont.cuba.gui.WindowParam
+import com.haulmont.cuba.gui.components.*
+import com.haulmont.cuba.gui.components.actions.ExcelAction
+import com.haulmont.cuba.gui.data.DsBuilder
+import com.haulmont.cuba.gui.data.impl.ValueCollectionDatasourceImpl
+import com.haulmont.cuba.gui.xml.layout.ComponentsFactory
+import de.diedavids.cuba.runtimediagnose.SqlConsoleSecurityException
+import de.diedavids.cuba.runtimediagnose.db.DbDiagnoseService
+import de.diedavids.cuba.runtimediagnose.db.DbQueryResult
 import de.diedavids.cuba.runtimediagnose.diagnose.DiagnoseExecutionFactory
 import de.diedavids.cuba.runtimediagnose.diagnose.DiagnoseType
 import de.diedavids.cuba.runtimediagnose.web.screens.diagnose.DiagnoseFileDownloader
-import groovy.transform.CompileStatic
 
 import javax.inject.Inject
 
-@CompileStatic
-abstract class ConsoleWindow extends AbstractWindow {
+class ConsoleWindow extends AbstractConsoleWindow {
 
-    public static final int SPLIT_POSITION_CENTER = 50
+    @Inject
+    DbDiagnoseService dbDiagnoseService
 
     @Inject
-    SourceCodeEditor console
+    ComponentsFactory componentsFactory
```

Comment:
are you still using it - or switched to `UiComponents` bean?

Reply:
I think it should be switched to `UiComponents` bean, `ComponentsFactory` is deprecated in CUBA 7.0